index.html

```diff
@@ -12,19 +12,31 @@
     <title>Naji portfolio</title>
 </head>
 
-<body>
+<body class="mobile">
     <header>
         <div class="navbar flex">
             <a class="logo" href="#"> My logo</a>
             <ul class="links">
-                <li><a href="#">Protfolio</a> </li>
-                <li><a href="#">About</a> </li>
-                <li><a href="#">Contact</a></li>
+                <li><a href="#Protfolio">Protfolio</a> </li>
+                <li onclick="mobileMenu()"><a href="#about">About</a> </li>
+                <li><a href="#Contact">Contact</a></li>
+            </ul>
+            <div class="menu"><i onclick="mobileMenu()" class="fa-solid fa-bars"></i></div>
+        </div>
+        <div class="mobile-menu non-display" id="mobile-menu">
```

Comment:
- [ ] Take a look at this record down here, guess what is happening? when the user scrolls this happens ⬇️ 
- The nav menu is not fixed, Please make a few changes to fix this. 

<img width="383" alt="2" src="https://user-images.githubusercontent.com/77843050/210264545-8593fa22-e102-43dd-a970-d79420b92e9e.png">


![5](https://user-images.githubusercontent.com/77843050/210264232-4053f8dd-8e00-4140-9708-c7bdbfa5629b.gif)